styleguide/components/21-atoms/spinner/_spinner.scss

```diff
@@ -0,0 +1,18 @@
+.spinner {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  div {
+    @include spot-image('spinner', 7rem);
+
+    text-indent: -200%;
```

Comment:
You might want to increase the text-indent, as seen in #597 
(Just in case...)

Reply:
I don't want it in px either.
Just in case someone comes along with a 10k monitor...

Reply:
Fine 😆That doesn't really make sense though. The percentage is relative to its parent, which means that when the spinner is used inside a `100px` container, a `-500%` value will still result in a `-5000px` offset.

(Just a thought, no need to change it again 😂)